### PR TITLE
Highlight downsides of toTitlecase

### PR DESCRIPTION
**Pull request by katrinleinweber**
This implements one possible resolution to our discussion in #243.

### CODE REVIEW

File: _extras/guide.md [hunk 1]
```diff
@@ -38,3 +38,10 @@ ____
 * If "https" doesn't work to fetch CrossRef during Advanced OpenRefine Functions, they can try "http"
 
 * If they need to diagnose failure to fetch the content from the URL they can check the "Store error" option in the "Add column by fetching URLs" dialogue and try looking at the common problems listed in the [documentation](https://docs.openrefine.org/manual/columnediting#common-errors)
+
+## Poweful transformations
+
+* In the titlecase exercise, highlight the fact that
+  each transformation can have unintended side effects,
+  and advise that running one cleanup operation too little
```
Comment (ostephens) [inline _extras/guide.md:46]: Minor grammar: this should be 'too few' rather than 'too little'

**Review by ostephens — CHANGES_REQUESTED** [review]
Apologies for the delay in reviewing this. Just a minor grammar changed needed in the instructor notes

File: _extras/guide.md (hunk 1 above)
Comment (katrinleinweber) [reply _extras/guide.md]: 😅 Thanks! Amended offline.

**Review by ostephens — CHANGES_REQUESTED** [review]

File: _extras/guide.md [hunk 2]
```diff
@@ -38,3 +38,10 @@ ____
 * If "https" doesn't work to fetch CrossRef during Advanced OpenRefine Functions, they can try "http"
 
 * If they need to diagnose failure to fetch the content from the URL they can check the "Store error" option in the "Add column by fetching URLs" dialogue and try looking at the common problems listed in the [documentation](https://docs.openrefine.org/manual/columnediting#common-errors)
+
+## Poweful transformations
+
+* In the titlecase exercise, highlight the fact that
+  each transformation can have unintended side effects,
+  and advise that running one cleanup operation too few
+  may sometimes be preferable to one too few.
```
[the remark this answers is not in the document]
Comment (katrinleinweber) [reply _extras/guide.md]: Ah, my apologies, @ostephens! I got confused there. Fixed now & resolved the merge conflict with #256.

**Pull request merged**